.drone.yml

```diff
@@ -407,6 +407,72 @@ steps:
     depends_on:
       - build
 
+---
+kind: pipeline
+type: docker
+name: testing-e2e
+
+platform:
+  os: linux
+  arch: amd64
+
+#depends_on:
+#  - testing-amd64
+#  - testing-arm64
+
+trigger:
+  event:
+    - push
+    - tag
```

Comment:
we definitely don't need this for tag, I would say we also don't need that for push, we already will have tested latest head with changes in PR, so why to run same tests with same code again? Tag and push ci jobs are already taking much time and resources